packages/chat-client/src/client.ts

```diff
@@ -160,18 +160,40 @@ export class ChatClient extends IChatClient {
     }
   };
 
-  public getInvites: IChatClient["getInvites"] = () => {
+  public getInvites: IChatClient["getInvites"] = (params) => {
     try {
-      return this.chatInvites.map;
+      return this.chatInvites
+        .getAll(
+          params?.account
+            ? {
+                account: params.account,
+              }
+            : undefined
+        )
+        .reduce<Map<number, ChatClientTypes.Invite>>((inviteMap, invite) => {
+          inviteMap.set(invite.id || 0, invite);
```

Comment:
Don’t think defaulting to zero makes sense here, it’s effectively an error case right so I think it’s better to either:
* simply return on that iteration without setting anything 
* Throwing an error

This would currently overwrite to `0` again and again if there’s multiple occurrences.

Reply:
Will add an error case, makes more sense